packages/driver/package.json

```diff
@@ -54,6 +54,7 @@
     "errorhandler": "1.5.1",
     "eventemitter2": "6.4.2",
     "express": "4.17.1",
+    "is-ip": "4.0.0",
```

Comment:
is-ip is also used by one of our dependencies, they use version 3.1.0 so now our bundle will have a duplicate. I don't think it's a big deal, but we could go back to 3.1.0 🤷‍♂️